diff --git a/vst3sdk/cmake/modules/SMTG_PlatformToolset.cmake b/vst3sdk/cmake/modules/SMTG_PlatformToolset.cmake
index 49ff3102..c4239bf2 100644
--- a/vst3sdk/cmake/modules/SMTG_PlatformToolset.cmake
+++ b/vst3sdk/cmake/modules/SMTG_PlatformToolset.cmake
@@ -60,7 +60,7 @@ macro(smtg_setup_platform_toolset)
                     set(CMAKE_ANDROID_STL_TYPE c++_static)
                     link_libraries(dl)
                  else()
-                    link_libraries(stdc++fs pthread dl)
+                    link_libraries(pthread dl)
                 endif(ANDROID)
             endif(SMTG_MAC)
         endif(XCODE)
diff --git a/src/plugin/lv2/CMakeLists.txt b/src/plugin/lv2/CMakeLists.txt
index a651fdd..e834389 100644
--- a/src/plugin/lv2/CMakeLists.txt
+++ b/src/plugin/lv2/CMakeLists.txt
@@ -38,7 +38,6 @@ if (CMAKE_SYSTEM_NAME MATCHES Windows)
       "-static-libstdc++"
       "-static-libgcc")
 else()
-    target_link_libraries(geonkick_lv2 PRIVATE "-lstdc++fs")
     target_link_libraries(geonkick_lv2 PRIVATE "-lX11 -lsndfile -lrt -lm -ldl -lpthread -lcairo")
 endif()
 
@@ -82,7 +81,6 @@ if (CMAKE_SYSTEM_NAME MATCHES Windows)
       "-static-libstdc++"
       "-static-libgcc")
 else()
-  target_link_libraries(geonkick_single_lv2 PRIVATE "-lstdc++fs")
   target_link_libraries(geonkick_single_lv2 PRIVATE "-lX11 -lsndfile -lrt -lm -ldl -lpthread -lcairo")
 endif()
 
diff --git a/src/plugin/vst/CMakeLists.txt b/src/plugin/vst/CMakeLists.txt
index ba72ae8..26eb377 100755
--- a/src/plugin/vst/CMakeLists.txt
+++ b/src/plugin/vst/CMakeLists.txt
@@ -80,7 +80,7 @@ target_link_libraries(${target} PRIVATE
       "-static-libgcc"
       )
 else()
-      target_link_libraries(${target} PRIVATE "-lstdc++ -lstdc++fs -lsndfile -lcairo -lm -lpthread")
+      target_link_libraries(${target} PRIVATE "-lstdc++ -lsndfile -lcairo -lm -lpthread")
 endif()
 
 add_dependencies(${target} geonkick_common)
diff --git a/src/standalone/CMakeLists.txt b/src/standalone/CMakeLists.txt
index df8b59c..54aece4 100755
--- a/src/standalone/CMakeLists.txt
+++ b/src/standalone/CMakeLists.txt
@@ -11,7 +11,6 @@ target_link_libraries(geonkick  geonkick_common api_standalone redkite)
 if (CMAKE_SYSTEM_NAME MATCHES Windows)
     target_link_libraries(geonkick "-mwindows -lstdc++ -lstdc++fs -lsndfile -lm -lpthread")
 else()
-    target_link_libraries(geonkick "-lstdc++fs")
     target_link_libraries(geonkick "-lX11 -ljack -lsndfile -lrt -lm -ldl -lpthread -lcairo")
 endif()
 
