--- a/plugin/lv2/CMakeLists.txt	2020-05-13 13:19:31.000000000 -0400
+++ b/plugin/lv2/CMakeLists.txt	2020-05-24 16:45:05.138976427 -0400
@@ -6,7 +6,6 @@
   ${GKICK_LV2_SOURCES})
 
 target_link_libraries(geonkick_lv2  geonkick_common api_plugin)
-target_link_libraries(geonkick_lv2 "-lstdc++fs")
 target_link_libraries(geonkick_lv2 "-lredkite -lX11 -lsndfile -lrt -lm -ldl -lpthread -lcairo")
 add_dependencies(geonkick_lv2 api_plugin)
 